src/components/Confirm/CaptureViewer/style.scss

```diff
@@ -30,36 +37,27 @@
   position: static;
 }
 
-.videoWrapper {
-  margin-bottom: 64*@unit;
-  flex: 1;
-}
-
 .pdfWrapper {
   display: block;     /* makes the containing PDF viewer width shrink correctly on window resize */
   overflow: overlay;  /* append scrollbars if necessary and overlay over/above content (only for Chrome, to avoid double scrollbars) */
 }
 
+.videoWrapper {
+  margin-bottom: 64*$unit;
+  flex: 1;
+}
+
```

Comment:
Minor: why did this move?

Reply:
No reason - probably when fiddling that wrapper around got moved 🙈